shared/templates/grub2_bootloader_argument_absent/bash.template

```diff
@@ -3,4 +3,9 @@
    See the OVAL template for more comments.
    Product-specific categorization should be synced across all template content types
 -#}}
+if {{{ bash_bootc_build() }}} ; then
+    sed -i -E "/kargs\s*=\s*\[\s*\"{{{ ARG_NAME }}}=[^\"]*\"\s*]/d" "$KARGS_DIR/*.toml"
```

Comment:
I would update the first line so it also deletes following `match-architectures` line if it is found (and also count with empty lines between `kargs` and `match-architectures`:
```
sed -i -E "/kargs\s*=\s*\[\s*\"{{{ ARG_NAME }}}=[^\"]*\"\s*]/{:a;N;/^\n$/ba;N;/match-architectures.*/d;}" "$KARGS_DIR/*.toml"
```
The second sed command should take care of just removing the specified kernel argument among other arguments, but as it is now it won't remove the comma character if our argument is followed by another argument or if it is preceded by another argument. Therefore, I would update the command like this:
```
sed -i -E -e "s/^(\s*kargs\s*=\s*\[.*)\"{{{ ARG_NAME }}}=[^\"]*\"[,[:space:]]*(.*]\s*)/\1\2/" -e "s/^(\s*kargs.*),[[:space:]]*\]$/\1\]/" "$KARGS_DIR/*.toml"
```